console/src/components/Main/utils.ts

```diff
@@ -47,9 +27,46 @@ const getProClickState = () => {
   }
 };
 
-export {
-  getLoveConsentState,
-  setLoveConsentState,
-  getProClickState,
-  setProClickState,
-};
+const MOUSEDOWN = 'mousedown';
+const TOUCHSTART = 'touchstart';
+type HandledEvents = [typeof MOUSEDOWN, typeof TOUCHSTART];
+type HandledEventsType = HandledEvents[number];
+type PossibleEvent = {
+  [Type in HandledEventsType]: HTMLElementEventMap[Type];
+}[HandledEventsType];
+type Handler = (event: PossibleEvent) => void;
+// Hook
+function useOnClickOutside(
+  refs: Array<React.RefObject<HTMLElement>>,
```

Comment:
@beerose made this change to handle multiple elements instead of just one. I hope this change is okay. Handling only one ref, caused a bug on the LoveSection components behaviour and hence I had to incorporate this change.